app/models/spotlight/resource.rb

```diff
@@ -74,13 +90,9 @@ def needs_provider?
       type.blank?
     end
 
-    def save_and_commit
+    def save_and_index
       save.tap do
```

Comment:
I confess #tap is new to me, and this line is confusing to me.  "save" would be save an ActiveModel object to a database, but now it has .tap on the end?